hydra-redis/test/querying_test.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import urllib.request
+import json
+from hydrus.hydraspec import doc_maker
+from os import sys, path
+
+PARENT_DIR = path.dirname(path.dirname(path.abspath(__file__)))
+sys.path.append(PARENT_DIR)
+import querying_mechanism
+
+
+class Tests:
+
+    def endpointTest(self):
+        """Test for all the endpoints class + collection endpoints"""
+        query = "show endpoints"
+        data = query_facades.user_query(query)
+        if len(data) == 11:
```

Comment:
You need to use assert statements in tests. Also, don't just compare the length you need to test data as well.

Reply:
I think we don't need to test for all the endpoint because we have two different test for class endpoints and collection endpoints. So, These two can cover this endpoint test. And Yeah, I have change comparison of length to comparison of data. I'll push it early.